CHANGELOG.md

```diff
@@ -2,6 +2,24 @@ faucet-pipeline-core version history
 ====================================
 
 
+v2.0.1
+------
+
+_2025-02-01_
+
+notable changes for end users:
+
+* dropped support for obsolete Node versions; now requires Node v18 or later
+* reduced number of dependencies
```

Comment:
Also, I opted not to list "updated dependencies" as a separate point here; seems implied?

Reply:
Fine with me :+1: